lib/base/utility.cpp

```diff
@@ -1946,3 +1938,20 @@ String Utility::GetIcingaDataPath(void)
 }
 
 #endif /* _WIN32 */
+
+void Utility::RenameFile(const String& source, const String& destination)
+{
+#ifdef _WIN32
+	if (remove(destination.CStr()) && errno != ENOENT)
```

Comment:
Also, more importantly you're unconditionally removing the target file, even if the source file doesn't exist.